ui/contexts/identity/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useContext, useEffect } from 'react';
+import { useAccountSyncing } from '../../hooks/identity/useProfileSyncing';
+
+const MetamaskIdentityContext = createContext(undefined);
+
+export const useMetamaskIdentityContext = () => {
+  const context = useContext(MetamaskIdentityContext);
+  if (!context) {
```

Comment:
What does context contain? Looking at the provider, it seems like this will always be undefined.

Lets remove the context and just return the provider component.

```tsx
// Recommend a doc comment to explain what this provider will do?
/**
* This Provider is used to initialise/run the syncing hooks.
*/
export const MetamaskIdentityProvider: React.FC = ({ children }) => {
  const {} = useAccountSyncing(...)
  useEffect(...)
  return <>{children}</>
}
```

Or we can move the useEffect to the `App.tsx` level (or similar).

```tsx
// In App.tsx or a similar root level component
import { useAccountSyncingEffect } from 'useProfileSyncingEffects'

// A location where devs can run their app effects, with comments
function useAppEffects() {
  ...
  useAccountSyncingEffect()
  ...
}

export const App = () => {
  useAppEffects();
  
  return (<></>)
}
```

Reply:
The home component is class based so unfortunately, no hooks land 😭 
I'll remove the context though, good point!

Reply:
Fair enough, TY for investigating!